src/components/Lookup/Lookup.test.js

```diff
@@ -5,21 +5,28 @@ import Lookup from './Lookup';
 
 it('renders without crashing', () => {
   const dictionary = {
-    dictionary: {},
-    exact: {},
-    greek: {},
-    latin: {},
+    diacriticLookup: {},
+    headwords: [],
+    lookup: {},
+    notes: [],
+    macronLookup: {},
+    roots: [],
   };
 
   const history = { push() { } };
   const match = { params: { word: 'hello' } };
 
+
+  const location = { search: 'here' };
+
+
```

Comment:
Let's get rid of all these extra spaces. Starting on line 16, the code should look like this:

```jsx
const history = { push: () => {} };
const match = { params: { word: 'hello' } };
const location = { search: 'here' };

const div = window.document.createElement('div');
```